backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5637900_sys_gh11302_process_remove_old_hu.sql

```diff
@@ -0,0 +1,131 @@
+INSERT INTO m_hu (ad_client_id, ad_org_id, created, createdby, isactive, m_hu_id,m_hu_pi_version_id, value, updated, updatedby, hustatus)
+VALUES (1000000,1000000,date('2022-05-05'),100,'Y',100,101,100.,date('2022-05-05'),100,'A');
+
+CREATE OR REPLACE FUNCTION remove_hu_entries_between_dates(
+    months NUMERIC,
+    placeholder NUMERIC
+)
+    RETURNS VOID
+    LANGUAGE plpgsql
+AS
+$$
+BEGIN
+    CREATE TEMP TABLE temp_m_hu AS
+    SELECT m_hu_id FROM m_hu WHERE hustatus != 'A' AND updated <= now() - (months || ' month')::INTERVAL LIMIT 10;
+    CREATE TEMP TABLE temp_m_hu_item AS
+    SELECT m_hu_item_id FROM m_hu_item WHERE m_hu_id IN (SELECT m_hu_id FROM temp_m_hu);
+    CREATE TEMP TABLE temp_m_inventoryline AS
+    SELECT m_inventoryline_id FROM m_inventoryline WHERE m_hu_id IN (SELECT m_hu_id FROM temp_m_hu);
+
+    -- No FK-Constrains
```

Comment:
not sure what this comment as well as "One FK-Constrains" further down actually mean, or how they help the reader of this fucntion